pkg/csi/cinder/nodeserver_test.go

```diff
@@ -380,3 +383,19 @@ func TestNodeExpandVolume(t *testing.T) {
 	assert.Equal(expectedRes, actualRes)
 
 }
+
+func TestNodeGetVolumeStats(t *testing.T) {
```

Comment:
what about the success case? could we add test for that as well?

Reply:
Success cases are done.